packages/snaps-sdk/src/ui/components/panel.ts

```diff
@@ -91,6 +94,9 @@ export const ComponentStruct = union([
   TextStruct,
   RowStruct,
   AddressStruct,
+  InputStruct,
+  FormStruct,
```

Comment:
Should this only be `FormStruct`? Do we allow stray inputs and buttons?

Reply:
Yes we allow stray inputs and buttons, buttons can be used to trigger other things than form submission and inputs can be read without the need of a form submission too.

Reply:
The code I proposed for merging the state yesterday assumes that inputs are contained inside forms. So we need to either change that or disallow inputs outside of forms

Reply:
I already changed that in the implementation you proposed